package.json

```diff
@@ -45,8 +45,10 @@
     "@vibrant/color": "^3.2.1-alpha.1",
     "@vibrant/generator-default": "^3.2.1-alpha.1",
     "@vibrant/quantizer-mmcq": "^3.2.1-alpha.1",
+    "buffer": "^6.0.3",
     "id3-parser": "^2.0.0",
     "idb-keyval": "^5.0.6",
+    "music-metadata": "^7.11.3",
```

Comment:
You could possibly use import [`music-metadata-browser`](https://github.com/Borewit/music-metadata-browser) and pass [`Response.blob()`](https://developer.mozilla.org/en-US/docs/Web/API/Response/blob) to [`parseblob()`](https://github.com/Borewit/music-metadata-browser#parseblob-function)

Reply:
> I have 3554 audio files (I have 3556 for real, but the Chrome directory API for some reason leaves two out ¯_(ツ)_/¯) btw, and most of them are FLAC, so I guess for most people this shouldn't be a problem.

Make sure parsing is not done in parallel.

FLAC should be not problem. In case you are tranfering the data via HTTP, this can be optimed via a solution like [@tokenizer/s3](https://github.com/Borewit/tokenizer-s3) which is only retrieving the parts reqired and not the entire file. But it makes things more complex.

Reply:
`parseBlob` uses `FileReader` internally to convert blob to Uint8Array and pass that to`parseBuffer` (sorry I know that you know how it works). Given that we can do the same thing with 2 lines of code, I don't think it's worth importing it just because of that.